CHANGELOG.md

```diff
@@ -14,15 +14,3 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ### Deprecated
 - none
```

Comment:
what about the other stuff? We don't use this `none` convention. Also, why not outline what the SDK already has right now? Unless it has nothing.

Reply:
this is just a template for now. I am not preparing for the release, so this is no concern of mine